From c4fa936a09a41f1e1b79f466ebc6321f2c0e6c35 Mon Sep 17 00:00:00 2001
From: Bob Beck <bbe@google.com>
Date: Thu, 22 Jun 2023 08:52:00 -0600
Subject: [PATCH 1/2] Include <vector> in parse_values.cc

This is used here and is currently obtained via hopes and prayers
from other dependencies. It it needed when built outside of chrome.
---
 net/der/parse_values.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/net/der/parse_values.cc b/net/der/parse_values.cc
index 6fa96d806cf66..6ff5024393212 100644
--- a/net/der/parse_values.cc
+++ b/net/der/parse_values.cc
@@ -5,6 +5,7 @@
 #include "net/der/parse_values.h"
 
 #include <tuple>
+#include <vector>
 
 #include "base/check_op.h"
 #include "base/notreached.h"
-- 
2.41.0.162.gfafddb0af9-goog

